give.php

```diff
@@ -614,6 +614,9 @@ private function includes() {
 			// Include API
 			require_once GIVE_PLUGIN_DIR . 'src/API/API.php';
 
+			// Include Onboarding
+			require_once GIVE_PLUGIN_DIR . 'src/Onboarding/Onboarding.php';
```

Comment:
Please _do not_ include files from `src` as they should be auto-loaded. If you need to register or boot something use a Service Provider.

Reply:
Yeah, so, I do plan to refactor to that, but these files are, like, a week old and follow the conventions that I saw in the codebase at that time. If I can get the changes in, then I can refactor.